handler/src/main/java/io/netty/handler/ssl/OpenSslSessionContext.java

```diff
@@ -60,6 +63,20 @@ public void setTicketKeys(byte[] keys) {
     }
 
     /**
+     * Sets the SSL session ticket keys of this context.
+     */
+    public void setTicketKeys(OpenSslSessionTicketKey... keys) {
+        if (keys == null) {
+            throw new NullPointerException("key");
+        }
+        SessionTicketKey[] ticketKeys = new SessionTicketKey[keys.length];
+        for (int i = 0; i < ticketKeys.length; i++) {
+            ticketKeys[i] = keys[i].key;
+        }
+        SSLContext.setSessionTicketKeys(context, ticketKeys);
```

Comment:
No... I think we should not expose tcnative classes to the enduser as it is an implementation detail